rules_poetry/defs.bzl

```diff
@@ -11,6 +11,7 @@ def deterministic_env():
         "PYTHONDONTWRITEBYTECODE": "1",
         "PYTHONHASHSEED": "0",
         "SOURCE_DATE_EPOCH": "315532800",  # set wheel timestamps to 1980-01-01T00:00:00Z
+        "USERPROFILE":".",
```

Comment:
Alternatively it seems we could forgo this method based on the comment on line 5, I dont really understand the backwards compat here? This function seems like it would meant to be internal use only, but thats just my assumption

Reply:
I briefly looked into it, I think it was alluding to using rules_python pip_install instead of the pip_install rule defined in the repo. But that rule seems be going away. rules_python is tending towards just passing locked requirements file as the sole strategy to install deps. This wont work for use with poetry until poetry can generate locked requirements files: https://github.com/python-poetry/poetry-plugin-export/issues/186

Reply:
You're referring to the note in https://rules-python.readthedocs.io/en/latest/pypi-dependencies.html#pip-rules right? Yeah that doesn't sound good. Do you think the pip_install defined in this repo will have compatibility issues with rules_python in the future as well?